modules/Microsoft.ServiceBus/namespaces/readme.md

```diff
@@ -496,6 +532,15 @@ module namespaces './Microsoft.ServiceBus/namespaces/deploy.bicep' = {
                     "service": "namespace"
                 }
             ]
+        },
```

Comment:
readme needs to be regenerated after latest changes where we removed these settings from the parameter file